spec/features/idv/doc_auth/document_capture_step_spec.rb

```diff
@@ -71,6 +71,7 @@
           result: 'Passed',
           billed: true,
         )
+        expect_costing_for_document
```

Comment:
thanks for adding!

Reply:
thanks for pointing out the regression tests!